addons/overheating/stringtable.xml

```diff
@@ -144,7 +144,25 @@
             <Chinese>過熱的武器將會有打不準和減少射擊初速的情況。適用於所有玩家</Chinese>
             <Hungarian>A túlmelegedett fegyverek kevésbé lesznek pontosak és csökkent a lövés sebessége. Minden játékosra vonatkozik.</Hungarian>
         </Key>
-        <Key ID="STR_ACE_Overheating_unJamOnreload_displayName">
+        <Key ID="STR_ACE_Overheating_particleEffectsAndDispersionDistance_displayName">
+            <English>Distance for Effects and Dispersion</English>
```

Comment:
```suggestion
            <English>Effects and Dispersion Distance</English>
```

Reply:
Putting distance at the start of the string implies it's the distance for both effects and dispersion, putting distance at the end can imply the distance is only for the dispersion, which I thought could confuse people.

Reply:
Both versions are fine and the suggestion is mostly how I would write it. I guess we can leave this one as is.